compose/docker-compose.yml

```diff
@@ -53,6 +78,11 @@ services:
       - "5672:5672"
     volumes:
       - rabbitmqdata:/var/lib/rabbitmq
+    healthcheck:
+      test: 'rabbitmq-diagnostics -q ping'
+      interval: 30s
```

Comment:
The healthcheck params are repeated a few times, can we make this into a YAML anchor like volumes are done?

Reply:
Remove all interval, timeout, and start-period, and use default value.

Reply:
👍